### ISSUE TITLE
haiku theme math centering doesn't work

### ISSUE THREAD

**Issue by hyperbolicTom** [sphinx-doc/sphinx#2294]
I'm using Sphinx 1.2.3 on Fedora 23. I'm using the Haiku theme.
It doesn't center .. math:: properly. The bug is a clash with basic.css. That uses div.body. This is the fix to Haiku, but maybe the bug is in basic.css?

```
--- /usr/share/sphinx/themes/haiku/static/haiku.css_t   2014-05-02 23:52:31.000000000 -0400
+++ haiku.css_t 2016-02-02 08:43:44.537037240 -0500
@@ -369,3 +369,7 @@
   margin: -1px -10px;
   padding: 0 12px;
 }
+
+div.math p {
+ text-align: center;
+}
```

**Comment by tk0miya** [comment]
In my short investigation:
`haiku` theme overrides the style by following declaration:

```
p {
    text-align: justify;
}
```

And more, `haiku` theme do not generates `<div class="body">`, so original defintion in basic.css do not effect if not overrided.

So we should redefine new style for `haiku` theme in `haiku.css_t` like suggested above.

In addition, I found some themes do not do centering the equations:
- agogo
- scrolls
- traditional

**Comment by tk0miya** [comment]
Fixed at 18c6a6a.
Please try it :-)
